src/CMakeLists.txt

```diff
@@ -91,6 +91,13 @@ if(REDIS_STORAGE_BACKEND)
     PUBLIC standard_settings standard_warnings HIREDIS::HIREDIS
            third_party_lib)
 endif()
+if(REDISS_STORAGE_BACKEND)
+  target_compile_definitions(ccache_lib PRIVATE -DHAVE_REDISS_STORAGE_BACKEND)
+  target_link_libraries(
+    ccache_lib
+    PUBLIC standard_settings standard_warnings HIREDIS::HIREDIS HIREDIS::HIREDIS_SSL
```

Comment:
The standard* and third party libraries are already added elsewhere.

The same applies to the redis backend.

Reply:
It is theoretically possible to disable redis, but enable rediss. The rest was just copy/paste, I guess

Reply:
I meant, the `REDIS_STORAGE_BACKEND` block above also links to standard and third party libs.

Reply:
It should be fixed, also it was calling the tests always. But that's not in _this_ backend